app/controllers/bulk_update/trainees/uploads_controller.rb

```diff
@@ -27,6 +27,7 @@ def create
         if @bulk_add_trainee_upload_form.valid?
           # TODO: Dry run method
           upload = @bulk_add_trainee_upload_form.save
+          SendCsvSubmittedForProcessingEmailService.call(user: current_user, upload: upload)
```

Comment:
At this point we validate the CSV to see if it is ready for submission and thus trainee creation. If the intention is to submit the email once we start creating trainees from the CSV file this is initiated at [BulkUpdate::Trainees::SubmissionsController#create](https://github.com/DFE-Digital/register-trainee-teachers/blob/feature/7785-create-confirmation-email-and-trigger-sending/app/controllers/bulk_update/trainees/submissions_controller.rb#L15)